buildAndReleaseTask/tests/create_stack_if_not_found.ts

```diff
@@ -0,0 +1,123 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
```

Comment:
This unit test, like the other ones in this folder, has a lot of repetition. I have created a new issue to address that separately.